### PR TITLE
Updated vehicles marks and models specs

### LINKED ISSUE #113: Обновить спецификации марок и моделей Транспортных средств
Обновить спецификации марок и моделей Транспортных средств. 
Добавить новые ТС.

### PR DESCRIPTION
**Pull request by Zaboday**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes

## Description

Updated vehicles marks and models specs.

Closes #113 

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in [CHANGELOG.md](https://github.com/avtocod/specs/blob/master/CHANGELOG.md) file

### CODE REVIEW

File: vehicles/default/models_municipal.json [hunk 1]
```diff
@@ -2,1406 +2,2017 @@
     {
         "name": "100",
         "id": "ID_MARK_PISTENBULLY_MODEL_100",
-        "mark_id": "ID_MARK_PISTENBULLY"
+        "mark_id": "ID_MARK_PISTENBULLY",
+        "vehicle_type": "ID_TYPE_MUNICIPAL"
```
Comment (Reallife) [inline vehicles/default/models_municipal.json]: No need to add `vehicle_type` field. All models already separated by type. In each file presents models for only one vihicle type.

Comment (Zaboday) [reply vehicles/default/models_municipal.json]: Remoced `vehicle_type` from all models files.

File: vehicles/default/models_bus.json [hunk 2]
```diff
@@ -224,41 +349,6 @@
         "id": "ID_MARK_KAVZ_MODEL_3976",
         "mark_id": "ID_MARK_KAVZ"
     },
-    {
```
Comment (Reallife) [inline vehicles/default/models_bus.json]: Try to decrease deletions count. Maybe add new entries at the and of file

File: vehicles/default/models_trailer.json [hunk 3]
```diff
@@ -9448,5 +9448,1600 @@
...
+        "id": "ID_MARK_BEST_MODEL_PRESTIZH_2_0",
+        "mark_id": "ID_MARK_BEST"
+    },
+    {
+        "name": "Прицеп для снегохода",
+        "id": "ID_MARK_BEST_MODEL_PRITSEP_DLYA_SNEGOKHODA",
+        "mark_id": "ID_MARK_BEST"
+    },
+    {
+        "name": "Цинк Мастер 1,7",
```
Comment (tarampampam) [inline vehicles/default/models_trailer.json]: Точно **запятые** в названии?

File: vehicles/default/models.json [hunk 4]
```diff
@@ -13633,5 +13633,1895 @@
...
+        "id": "ID_MARK_DESOTO_MODEL_CUSTOM",
+        "mark_id": "ID_MARK_DESOTO"
+    },
+    {
+        "name": "Longchamp",
+        "id": "ID_MARK_DE_TOMASO_MODEL_LONGCHAMP",
+        "mark_id": "ID_MARK_DE_TOMASO"
+    },
+    {
+        "name": "3=6",
```
Comment (tarampampam) [inline vehicles/default/models.json]: Именно `=`?

File: vehicles/default/models_trailer.json (hunk 3 above)
Comment (Reallife) [reply vehicles/default/models_trailer.json]: @tarampampam Да. https://auto.ru/trailer/best/zink_master_1_7/all/

File: vehicles/default/models.json (hunk 4 above)
Comment (Reallife) [reply vehicles/default/models.json]: @tarampampam Да. https://auto.ru/cars/dkw/3_6/all/

**Pull request merged**